app/e2e/tests/landing/landing-carousel.spec.ts

```diff
@@ -1,23 +1,67 @@
+import { PlaywrightLandingPage } from '@local/common/pages/playwright-landing-page';
 
 test.describe('carousel buttons', () => {
-    test('views carousel buttons next and back should go from participant view <-> moderator view', async ({ page }) => {
-        await page.goto('/');
-        await page.locator('text=Next').first().click();
-        await expect(page.locator('#page div:has-text("Moderator ViewHomeDashboardAbout UsLLorem IpsumTown Hall MeetingConcerns within ")').nth(3)).toBeVisible();
-        await page.locator('text=Back').nth(2).click();
-        await expect(page.locator('#page div:has-text("Participant ViewHomeDashboardAbout UsLLorem IpsumTown Hall MeetingConcerns withi")').nth(3)).toBeVisible();
+    test.use({ storageState: undefined });
+    test("Top carousel's next and back buttons work", async ({ page }) => {
+        // Arrange
+        const landingPage = new PlaywrightLandingPage(page);
+        await landingPage.goto();
+        // Act + Assert
+        await landingPage.nextButtons.first().click();
+        await expect(
+            page
+                .locator(
+                    '#page div:has-text("Moderator ViewHomeDashboardAbout UsLLorem IpsumTown Hall MeetingConcerns within ")'
+                )
+                .nth(3)
+        ).toBeVisible();
+        await landingPage.backButtons.nth(2).click();
+        await expect(
+            page
+                .locator(
+                    '#page div:has-text("Participant ViewHomeDashboardAbout UsLLorem IpsumTown Hall MeetingConcerns withi")'
+                )
+                .nth(3)
+        ).toBeVisible();
     });
 
-    test('roles carousel buttons next & back should go from participant role <-> moderator role <-> speaker role', async ({ page }) => {
-        await page.goto('/');
-        await page.locator('text=Next').nth(1).click();
-        await expect(page.locator('#page div:has-text("Moderator RoleThe mediators who handle participants\' questions to be answered by")').nth(3)).toBeVisible();
-        await page.locator('text=Next').nth(2).click();
-        await expect(page.locator('#page div:has-text("Speaker RoleThe main speaker of a discussion. The speaker does not see the quest")').nth(3)).toBeVisible();
-        await page.locator('text=Back').nth(3).click();
-        await expect(page.locator('#page div:has-text("Moderator RoleThe mediators who handle participants\' questions to be answered by")').nth(3)).toBeVisible();
-        await page.locator('text=Back').nth(3).click();
-        await expect(page.locator('#page div:has-text("Participant RoleThe residents who want to engage in discussion on a policy topic.")').nth(3)).toBeVisible();
+    test("Bottom carousel's next and back buttons work", async ({ page }) => {
+        // Arrange
+        const landingPage = new PlaywrightLandingPage(page);
+        await landingPage.goto();
+        // Act + Assert
+        await landingPage.nextButtons.nth(1).click();
```

Comment:
I think using nth here and below is going to be pretty frustrating long term? Am I correct in interpreting that any change to the order is going to break tests, but that doesn't mean it's broken for the user?

I think tests should instead focus primarily on making sure the experience for the user is not broken. Not that minor things like order change.

Reply:
I agree these tests are ugly, I was mainly just doing this for more examples of using this pattern. I could remove them or add a TODO that they will be removed. The UI/UX team is currently working on an updated landing page design so these tests will be replaced once the new page is finalized.